magento/pwa-studio
# Order History Page Search

## Description

Added feature to search by order number on the order history page.

## Related Issue
Closes [PWA-888](https://jira.corp.magento.com/browse/PWA-888)

## Acceptance
Should be able to search by order number on the order history page.

### Verification Stakeholders
@schensley 
@dpatil-magento 

### Verification Steps
1. Go to the deployed PR's order history page.
2. Enter a valid order number in the search field. You should be able to see the order details of the given order number.
3. Enter an invalid number and you should see a message stating there are no orders yet.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/102806794-21191200-4383-11eb-989c-c3e41701b690.png)

![image](https://user-images.githubusercontent.com/35203638/102806725-05157080-4383-11eb-9443-1e6b8f527a00.png)

![image](https://user-images.githubusercontent.com/35203638/102816920-6c87ec00-4394-11eb-9d4e-22fc60c9b7df.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
